src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -127,7 +127,9 @@
     private static final int MAX_AVATAR_SIZE = 16384;
 
     private static final Logger LOGGER = Logger.getLogger(BitbucketServerAPIClient.class.getName());
-    private static final String API_BASE_PATH = "/rest/api/1.0";
+    private static final String BASE_PATH = "/rest/%s/1.0";
```

Comment:
No thanks to string formatting, please use the existing library handyuri or please revert this change.